### PR TITLE
Donut/rename group

### PR DESCRIPTION
**Pull request by amitsha**

### CODE REVIEW

**Comment by Hamo69** [comment]
The Donut image uses the Powered By MagenTys logo. This text should be removed.

File: README.md [hunk 1]
```diff
@@ -174,4 +174,4 @@ We currently have plans to support:
 
 This project is under an MIT license
 
-Powered by: [MagenTys](http://magentys.io)
+Powered by: [MagenTys](http://magentys.io) [MechanicalRock](https://www.mechanicalrock.io/#/)
```
Comment (Hamo69) [inline README.md]: can we add an & or an AND in between the two companies.
The MagenTys site should also be https

File: build.sbt [hunk 2]
```diff
@@ -4,9 +4,9 @@ import scala.collection.JavaConverters._
 
 name := "donut"
 
-organization := "io.magentys"
+organization := "report.donut"
 
-description := "Magentys Donut - Reporting Tool"
+description := "Donut - Reporting Tool"
```
Comment (Hamo69) [inline build.sbt]: Donut - Automated Testing Reporting Tool
?

File: README.md [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+[![Build Status](https://travis-ci.org/DonutReport/donut.svg?branch=master)](https://travis-ci.org/DonutReport/donut)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/report.donut/donut/badge.svg)](https://maven-badges.herokuapp.com/maven-central/report.donut/donut)
+[![Javadoc](https://javadoc-emblem.rhcloud.com/doc/report.donut/donut/badge.svg)](http://www.javadoc.io/doc/report.donut/donut)
 
-Donut is an open-source framework by the [MagenTys](https://magentys.io) team which is designed to produce clear and concise test execution reports over your unit, integration and acceptance tests.
-Donut currently supports any tool that produces gherkin json (ie. cucumber-jvm etc). For other frameworks like SpecFlow, JUnit, NUnit, you can use the adapters listed below to generate the gherkin jsons, which can then be processed by **donut**.
+Donut is an open-source framework by [MagenTys](https://magentys.io) & [MechanicalRock](https://mechanicalrock.io) and is designed to produce clear and concise test execution reports of your unit, integration and acceptance tests.
```
Comment (Hamo69) [inline README.md]: Donut is an open-source framework by the teams at [MagenTys](https://magentys.io) & [MechanicalRock](https://mechanicalrock.io)

File: README.md (hunk 1 above)
Comment (Hamo69) [inline README.md]: should probably remove the #/ - so https://www.mechanicalrock.io/

File: README.md (hunk 3 above)
Comment (Hamo69) [reply README.md]: [Mechanical Rock](https://mechanicalrock.io)

File: README.md (hunk 1 above)
Comment (Hamo69) [reply README.md]: [Mechanical Rock](https://mechanicalrock.io)

Comment (Hamo69) [reply README.md]: [Mechanical Rock](https://mechanicalrock.io)

File: README.md (hunk 3 above)
Comment (amitsha) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (amitsha) [reply README.md]: Done

Comment (amitsha) [reply README.md]: Done

File: build.sbt (hunk 2 above)
Comment (amitsha) [reply build.sbt]: Done

**Pull request merged**